db/migrate/20190408195208_add_unique_index_to_export_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToExportRules < ActiveRecord::Migration[5.2]
```

Comment:
Okay to combine this migration into the migration file that creates the `export_rules` table, just to have fewer migrations?

Reply:
I would rather not do that because these are part of different commits. All the code regarding the addition of the unique index is isolated into one commit.

Reply:
👍